src/progress-indicator/overlay.scss

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-progress-indicator-overlay($backgroundColor: iui-color-background-1, $pointerEvent: auto) {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  z-index: 500;
+  pointer-events: $pointerEvent;
+
+  // Blur fallback
+  @include themed {
+    background-color: rgba(
+      t(#{$backgroundColor}-rgb),
+      t(iui-opacity-2)
+    );
+  }
+
+  // With blur
+  @supports(backdrop-filter: $iui-blur-filter) {
+    @include themed {
+      background-color: rgba(
+        t(#{$backgroundColor}-rgb),
+        t(iui-opacity-3)
+      );
+      backdrop-filter: $iui-blur-filter;
+    }
+  }
+
+  .iui-progress-indicator-linear {
+    width: 50%;
+    max-width: 33vw;
+  }
+
+  // Fade out on completion
+  @keyframes closeAnimation {
```

Comment:
This needs JS?

Reply:
How this animation is supposed to work? Dont think anyone will apply opacity not to show overlay...
I guess we could hack some appearing/disapearing animation on react side.

Reply:
For toast we are using CSS animation and when it ends, element is destroyed. It is handled by `react-transition-group`. So I think it is ok here as it fallows the same naming.